### ISSUE TITLE
Not in scope: type constructor or class 'LuaRef'

### ISSUE THREAD

**Issue by locallycompact** [neovimhaskell/nvim-hs#91]
Hi, I'm getting the following error trying to build the example.

```
nvim-hs                          >
nvim-hs                          > /run/user/1000/stack-a8df55139a77fd11/nvim-hs-2.1.0.5/library/Neovim/API/Text.hs:22:2: error:
nvim-hs                          >     Not in scope: type constructor or class ‘LuaRef’
nvim-hs                          >    |
nvim-hs                          > 22 | $(generateAPI textVectorTypeMap)
nvim-hs                          >    |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
nvim-hs                          >
nvim-hs                          > /run/user/1000/stack-a8df55139a77fd11/nvim-hs-2.1.0.5/library/Neovim/API/Text.hs:22:2: error:
nvim-hs                          >     Not in scope: type constructor or class ‘LuaRef’
nvim-hs                          >    |
nvim-hs                          > 22 | $(generateAPI textVectorTypeMap)
nvim-hs                          >    |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by saep** [comment]
That is the code generation of the neovim API. Could you please tell me your neovim version?

The code generator uses the output of `nvim --api-info` to generate Haskell bindings. I guess that something in that definition has changed and the code generation has to be extended.

**Comment by saep** [comment]
I'm surprisingly busy at the moment. I'll either find some motiviation and time on friday or the weekend after the next one.

**Comment by saep** [comment]
Fixed in version 2.1.0.7 via https://github.com/neovimhaskell/nvim-hs/commit/2b3b05167d4513de7c549b5b32df08ec7b2ec247
